wire-schema/src/main/java/com/squareup/wire/schema/Field.kt

```diff
@@ -125,7 +126,9 @@ class Field private constructor(
 
     val memberName = if (isExtension) qualifiedName else name
     val protoMember = ProtoMember.get(enclosingType, memberName)
-    if (!markSet.contains(protoMember)) return null
+
+    if (!markSet.contains(protoMember) &&
```

Comment:
maybe I should add the `(GOOGLE_PROTOBUF_OPTION_TYPES.contains(enclosingType)` exception within the `markSet#contains` method...